### PR TITLE
Config rework

### PR DESCRIPTION
**Pull request by Robin-Hoodie**
- rename `tFunctionNames` to `customTFunctionNames`
- Set `useI18nextDefaultValue` default value to `false`

Also made me realize #128 is no longer needed should this be ready to be merged in. 

If we want to hold off on merging this in for a while, #128  might be worth to be merged in for now however.

### CODE REVIEW

**Comment by gilbsgilbs** [comment]
Since this is a breaking change, I'll rather merge #128 and keep this PR open.

File: src/config.ts [hunk 1]
```diff
@@ -76,17 +76,14 @@ export function parseConfig(opts: Partial<Config>): Config {
       'i18next',
       'i18n',
     ]),
-    tFunctionNames: coalesce(opts.tFunctionNames, ['t']),
+    customTFunctionNames: coalesce(opts.customTFunctionNames, ['t']),
```
Comment (gilbsgilbs) [inline src/config.ts]: Note for v1: the default value here should be an empty array (to be consistent with other `custom*` options). However, we may want to allow user to disable default `t` function extraction (not sure if it's really useful).

**Comment by Robin-Hoodie** [comment]
> Since this is a breaking change, I'll rather merge #128 and keep this PR open.

So are there any changes you're requesting?

**Comment by xDisfigure** [comment]
> > Since this is a breaking change, I'll rather merge #128 and keep this PR open.
> 
> So are there any changes you're requesting?

Hello @Robin-Hoodie 
You can take a look to this feature request : https://github.com/gilbsgilbs/babel-plugin-i18next-extract/pull/168

**Comment by Robin-Hoodie** [comment]
@xDisfigure Haven't looked at this in forever. Am I understanding things correctly that this PR is no longer necessary with the merge of #168 ?

**Comment by Robin-Hoodie** [comment]
@xDisfigure Let's close this PR if it's no longer relevant

**Pull request closed without merging**